### PR TITLE
Add symbols to MSI

### PR DESCRIPTION
**Pull request by kevingosse**
Include the PDB symbols to the installer

### CODE REVIEW

**Comment by colin-higgins** [comment]
Those wix files are actually kept in sync with the PrepareRelease tool.
We should probably add a comment mentioning they are auto-generated.
image
https://github.com/DataDog/dd-trace-dotnet/blob/master/build/tools/PrepareRelease/Program.cs#L45
https://github.com/DataDog/dd-trace-dotnet/blob/master/build/tools/PrepareRelease/SyncMsiContent.cs

**Comment by lucaspimentel** [comment]
For the dlls we install in the GAC, we should copy the PDBs into `%WINDIR%\System32\Symbols\dll` (in addition to `Program Files` like you did here).

**Comment by kevingosse** [comment]
> For the dlls we install in the GAC, we should copy the PDBs into `%WINDIR%\System32\Symbols\dll` (in addition to `Program Files` like you did here).

I've considered it, but I'm not sure we should. I don't really like putting additional files into System32, and it doesn't bring much (the symbols are still available in the Datadog folder, the developer just has to load them manually).

Not to mention that the GAC is versioned and `%WINDIR%\System32\Symbols\dll` wouldn't be.

**Review by lucaspimentel — CHANGES_REQUESTED** [review]
See @colin-higgins's comment about the `PrepareRelease` tool: https://github.com/DataDog/dd-trace-dotnet/pull/1364#issuecomment-818742470

**Comment by kevingosse** [comment]
> Those wix files are actually kept in sync with the PrepareRelease tool.

I was wondering why nobody ever reformatted those files 😄

File: src/WindowsInstaller/Files.Managed.Net45.GAC.wxs [hunk 1]
```diff
@@ -11,18 +11,36 @@
               Source="$(var.TracerHomeDirectory)\net45\Datadog.Trace.AspNet.dll"
               KeyPath="yes" Checksum="yes" Assembly=".net"/>
       </Component>
+      <Component Win64="$(var.Win64)">
```
Comment (zacharycmontoya) [inline src/WindowsInstaller/Files.Managed.Net45.GAC.wxs:14]: Looks like we need additional logic...we don't want the pdb files to be added to this "Net45.GAC" file or the "Net461.GAC" file

Comment (kevingosse) [reply src/WindowsInstaller/Files.Managed.Net45.GAC.wxs:14]: ugh, good catch

Comment (kevingosse) [reply src/WindowsInstaller/Files.Managed.Net45.GAC.wxs]: Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1364/commits/fe760dd3272ceddde06de35493b7882e0bdd6dca

**Review by zacharycmontoya — APPROVED** [review]
LGTM good stuff!

**Review by lucaspimentel — APPROVED** [review]

**Pull request merged**